español/app/es.json

```diff
@@ -1,961 +1,961 @@
 {
-  "YES": "Yes",
+  "YES": “Sí”,
```

Comment:
Unfortunately the neutral quote marks ( " ) have been altered to the typographic marks ( ” ) in several places. These will all need to be replaced by " in order to commit these changes. Thanks!